tasks/selinux.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure the service and the ports status with the selinux role
+  include_role:
+    name: fedora.linux_system_roles.selinux
+  vars:
+    _cockpit_port: "{{ cockpit_port if cockpit_port is defined else 9090 }}"
+    selinux_ports: "{{ [{'ports': _cockpit_port, 'proto': 'tcp',
+                         'setype': 'websm_port_t',
+                         'state': 'present', 'local': 'true'}] }}"
+  when:
+    - cockpit_manage_selinux | bool
```

Comment:
I'd add an additional condition that `cockpit_port is defined` (and remove that from line 7), so that this isn't even invoked when the port doesn't get customized (the common case). Avoids traps and is faster.

Reply:
hmm - since `cockpit_port` is part of the public api of the role, it should be defined in `defaults/main.yml` as `null` - so check for `cockpit_port is not none` (`null` in YAML == `none` in Jinja)

Reply:
> hmm - since `cockpit_port` is part of the public api of the role, it should be defined in `defaults/main.yml` as `null` - so check for `cockpit_port is not none` (`null` in YAML == `none` in Jinja)

Is it ok for me to make this change in this pr?

Reply:
> > hmm - since `cockpit_port` is part of the public api of the role, it should be defined in `defaults/main.yml` as `null` - so check for `cockpit_port is not none` (`null` in YAML == `none` in Jinja)
> 
> Is it ok for me to make this change in this pr?

Yes, please.